input/pagecontent/StructureDefinition-au-erequesting-imaging-request-intro.md

```diff
@@ -0,0 +1,9 @@
+### Usage scenarios
+The following are supported usage scenarios for this profile:
+
+- Query for a request for a diagnostic imaging service for a patient
+- Record or update a request for a diagnostic imaging service for a patient
+- 
+
+### Profile specific implementation guidance
+- The use of coding can vary significantly across systems, implementers need to understand that they may encounter codes they do not recognise and be prepared to handle those resources appropriately. Systems **SHALL** populate `ServiceRequest.code.text` so that systems can at least display the service requested even if the system does not recognise the code supplied.
```

Comment:
Why is this bit of profile-specific implementation guidance here but not the guidance on the coding? I assume the PR on profile-specific guidance that was open would be where this would be handled after this PR is done.

If that isn't the plan, then close the other PR and move the guidance for pathology into this page.

Reply:
Moved profile implementation guidance from the other PR to here.